applications/GeoMechanicsApplication/tests/test_dynamic/test_constant_point_load_2d_axi/README.md

```diff
@@ -0,0 +1,22 @@
+# Test Cases for Dynamic solution
+
+**Author:** [Mohamed Nabi](https://github.com/mnabideltares)
+
+**Source files:** [Dynamic solution](https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/test_dynamic/test_constant_point_load_2d_axi)
+
+## Case Specification
+In this test case, a two-dimentional axisymmetric soil block with dimensions of 10 m x 10 m is considered. The pressure is fixed at 0 Pa throughout the entire simulation to eliminate the effect of water pressure.  This is done to allow comparison of our results with published semi-analytical solutions. The bottom side of the domain is fixed, while the sides are allowed to move only in the vertical direction. A sudden force of -1000 N is then applied in the vertical direction at the top surface of the block. The simulation spans 1 second. This test is conducted on a mesh with triangular elements of type 2D3N. The deformation at the surface is then compared with semi-analytical results.
+
+The geometry and boundary conditions are shown below:
+
+<img src="../documentation_data/test_constant_point_load_2d_axi_conditions.svg" alt="Visualization of the geometry and the boundary conditions" title="Visualization of the geometry and the boundary conditions" width="600">
+
+## Results
+
+The picture below illustrates the variation in vertical displacement over time for node 191, which is 3 meters away from the loading point. The results are compared with the semi-analytical solution.
```

Comment:
3 meters in which direction? You may consider to indicate it in the picture of the model.